api/planmill1_5.raml

```diff
@@ -139,6 +140,19 @@ traits:
                   "message": "Invalid date pattern",
                   "code": "00500"
                 }
+  - some:
```

Comment:
I don't see why its being modified or added. developer link has the same content right now.

Reply:
It is NOT the same at all. Compare understand it well. I will ignore this review comment.